src/checkers/periodicSecretChecker.go

```diff
@@ -86,6 +88,23 @@ func (p *PeriodicSecretChecker) StartChecking() {
 		}
 
 		for _, secret := range secrets {
+			include, exclude := false, false
```

Comment:
let's include a log line above this that indicates the secret is being checked at all.  it will help debugging when people are wondering why a particular secret is/is not being checked.

Reply:
Do you want this as an informational or Debug, logging is going to get extremely verbose for some of us who have 10,000+ helm releases. 

Helm3 in particular would do this
```
I0218 18:49:49.614623 1913389 periodicSecretChecker.go:104] Skipping secret sh.helm.release.v1.myapp.v167 in myapp because helm.sh/release.v1 is not included in your typelist
I0218 18:49:49.614630 1913389 periodicSecretChecker.go:104] Skipping secret sh.helm.release.v1.myapp.v168 in myapp because helm.sh/release.v1 is not included in your typelist
I0218 18:49:49.614637 1913389 periodicSecretChecker.go:104] Skipping secret sh.helm.release.v1.myapp.v169 in myapp because helm.sh/release.v1 is not included in your typelist
```

Reply:
Sorry, i didnt see this was glog, there is no debug option. 
i updated it to output the following. 
```
Skipping secret sh.helm.release.v1.myapp.v169 in myapp because helm.sh/release.v1 is not included in your secret-include-types [Opaque kubernetes.io/tls]
```

Should we be concerned about the log volume? It only runs hourly for most so that doesn't sound too bad.

Reply:
ugh, i hate glog so much.   one day i will move this over to something better.

yeah, i think b/c this only needs to check sparingly the log volume is fine.  i'm mainly concerned about the scenarios when someone files an issue "why isn't my secret being checked".  i need to see logs to know where it got filtered out to advise them.